<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>708</x>
      <y>9</y>
      <width>211</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>681</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="title">
     <string>Note</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport"/>
    <addaction name="actionDelete"/>
    <addaction name="actionExit"/>
    <addaction name="actionThrowToOtherStorage"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSort_by">
     <property name="title">
      <string>Sort by</string>
     </property>
     <addaction name="actionName"/>
     <addaction name="actionDate"/>
    </widget>
    <addaction name="menuSort_by"/>
   </widget>
   <widget class="QMenu" name="menuStorage">
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="title">
     <string>Storage</string>
    </property>
    <addaction name="actionSwitchStorage"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuStorage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save </string>
   </property>
   <property name="toolTip">
    <string>Save current note</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new note</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
  </action>
  <action name="actionDate">
   <property name="text">
    <string>Date</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export all</string>
   </property>
  </action>
  <action name="actionSwitchStorage">
   <property name="text">
    <string>Switch to archive</string>
   </property>
  </action>
  <action name="actionThrowToOtherStorage">
   <property name="text">
    <string>Throw to archive</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
